src/NzbDrone.Core/Localization/Core/en.json

```
@@ -83,6 +83,7 @@
   "Automatic": "Automatic",
   "AutomaticAdd": "Automatic Add",
   "AutomaticSearch": "Automatic Search",
+  "AutomaticUpdatesAreDisabledWhenUsingTheDockerUpdateMechanism": "Automatic updates are disabled when using the Docker update mechanism. Update the container image instead. Contact your container maintainer for additional details.",
```

> Contact your container maintainer for additional details.

What's the value in this? Seems like unnecessary noise.

---

userproof else they'd be coming to the sonarr discord?

---

Potentially, but now we're sending them to the maintainer's discord when it's not really something they need to deal with either. We already have docker disabled for updates and don't have many people asking how to update it.

```suggestion
  "AutomaticUpdatesAreDisabledWhenUsingTheDockerUpdateMechanism": "Automatic updates are disabled when using the Docker update mechanism. You will need to update the container image outside of Sonarr.",
```